### DISCUSSION TITLE
모바일의 뒤로가기 버튼을 통해 룸을 나가고 싶습니다.

### DISCUSSION THREAD

**Discussion by jun5640** [naverz/zepeto-studio-kor#312] [5. Q&A]
월드에서 룸을 나가는 기능을 직접 구현하고 싶습니다.

해당 기능을 지원해주는 API가 있나요?

Application.Quit() 작동을 안 합니다.

**Comment by hyeonjin-jo** [comment]
안녕하세요.

https://github.com/naverz/zepeto-multiplay-example/blob/main/Assets/ZepetoScripts/ClientStarter.ts

위의 스크립트에 있는 멤버 변수  private room: Room; 를 활용해서 룸 나가기 함수를 호출할 수 있습니다.

this.room.Leave();

**Reply by jun5640** [reply]
감사합니다!
